modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules.rules_defaults.inc

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Provide default rules integration to manage facebook instant article using API.
+ */
+
+/**
+ * Implements hook_default_rules_configuration().
+ */
+function fb_instant_articles_api_rules_default_rules_configuration() {
+  // Handling add/update node article as published in FB Instant Articles library.
+  $rule = rules_reaction_rule();
+  $rule->label = t('Add/update as published in FB Instant Articles library ' .
+                   'when a published Article node is saved');
+  $rule->active = FALSE;
+  $rule->event('node_presave')
+    ->condition('data_is', array('data:select' => 'node:status', 'value' => TRUE))
+    ->condition('data_is', array('data:select' => 'node:type', 'value' => 'article'))
```

Comment:
It would be nice if this value was not hardcoded to `article`, but relied on the `fb_instant_articles_display_is_article_type` function (I hope that's possible). I just discussed this with @scottrigby so he can maybe add more context if needed.